### ISSUE TITLE
Broken link found on H97 (Techniques for WCAG 2.1)

### ISSUE THREAD

**Issue by caztcha** [w3c/wcag#1913]
H97: Grouping related links using the nav element
https://www.w3.org/WAI/WCAG21/Techniques/html/H97

On the "Resources" section, there is one broken link found.
- HTML5 nav element

The correct link target should be as follows.
https://html.spec.whatwg.org/multipage/sections.html#the-nav-element

Please confirm and correct the link.

**Comment by alastc** [comment]
Thanks, I've [merged into a editorial fixes branch](https://github.com/w3c/wcag/commit/520bafa2c2e8e5af68020a9cd840d39f5aa78a23), which will go into main soon.
